src/app/public/modules/text-editor/text-editor.component.ts

```diff
@@ -210,24 +243,30 @@ export class SkyTextEditorComponent implements AfterViewInit, ControlValueAccess
     return this._value;
   }
 
-  public editorFocusStream = new Subject();
+  private _disabled: boolean = false;
```

Comment:
Our private prefixed `_` properties need to be grouped together and alphabetized. They also should appear last.

Reply:
fixed in "[fix order of private properties](https://github.com/blackbaud/skyux-text-editor/pull/21/commits/0294b52f18ffef9daed9c43f8fc0ae5252879522)"